CIP-1694/README.md

```diff
@@ -0,0 +1,504 @@
+
+### Treasury Withdrawals vs Project Catalyst
+
+Project Catalyst is currently one of the main drivers for withdrawals from the Cardano treasury. Each Catalyst round is usually followed by hundreds - if not thousands - of MIR requests to deliver funding to selected projects. In this new governance framework, however, we will only permit one trasury withdrawal per epoch.
+
+Since all withdrawal requests from the treasury must fit into a single transaction, this limits the number of projects that can be funded in a single epoch. If necessary, this can be worked around by e.g. splitting funding over multiple epochs, by transferring funds to a temporary holding pot, or by restricting the number of projects that are funded in each round.
+
+## Path to Active
+
+### Acceptance Criteria
+
+- [ ] A new ledger era is enabled on the Cardano mainnet, which implements the above specification. This will be split into two stages, as described above.
+
+### Implementation Plan
+
+This proposal restricts itself to the core on-chain mechanism. However, several issues remain to be discussed. These include:
+
+1. off-chain DRep expectations;
+2. bootstrapping the Constitutional Committee;
+3. the initial Constitution;
+4. DRep incentives;
+5. which values in the [ratification requirement table](#requirements) should be hard-coded, and which should be protocol parameters?
+
+Some of these issues may be addressed by changes to this document; others will need to be addressed in new CIPs or discussion documents.
+
+#### New Protocol Parameters
+
+New protocol parameters will be needed for the following:
+
+* the governance action deposit amount
+* governance action expiration (as a number of epochs from the current epoch)
+* thresholds for treasury withdrawals
+* each of the ratification thresholds
+
+As described above, some  of these will be updatable; others will be hard coded.
+
+// TODO: Decide on the initial parameter values and whether they should be updatable.
+
+In addition, the initial Constitutional Committee and the initial Constitution will need to be defined as part of a bootstrap process.
+
+#### Changes to the existing ledger rules
+
+* The `PPUP` transition rule will be rewritten and moved out of the `UTxO` rule and into the `LEDGER` rule as a new `TALLY` rule.
+
+  It will process the governance actions and the votes, ratify them, and stage governance actions for enactment in the current or next epoch, as appropriate.
+
+* The `NEWEPOCH` transition rule will be modified.
+* The `MIR` sub-rule will be removed.
+* A new `ENACTMENT` rule will be called immediately after the `EPOCH` rule. This rule will enact governance actions that have previously been ratified.
+* The `EPOCH` rule will no longer call the `NEWPP` sub-rule or compute whether the quorum is met on the PPUP state.
+
+#### More on DRep incentives
```

Comment:
Also: what about voter incentives? Are voters incentivised to delegate to DReps at all? That seems important. Otherwise I expect many people will simply not participate at all (or maybe that's fine?)

Reply:
The incentive is supposed to be that then they have a voice, but I get that you are concerned that this is maybe not enough. Has there ever been a government or some other body that has paid voters to vote?

Reply:
Well, several governments including Australia make it illegal not to vote, which is an incentive.

Reply:
> The incentive is supposed to be that then they have a voice, but I get that you are concerned that this is maybe not enough. Has there ever been a government or some other body that has paid voters to vote?

Legislators are paid by governments, in this systems it seems anyone can fill that role. And if someone acquires a large amount of delegation it seems prudent the blockchain should pay them so that they can treat their role as a full time job.

Reply:
And this is where you need legal procedure if you are to punish, otherwise that is dystopian. Forced political participation upholds established systems rather than allows for people to disengage from a broken system and begin to form new ones. It is anti-revolutionary and will stymie growth.

> Well, several governments including Australia make it illegal not to vote, which is an incentive.

Reply:
I'm not suggesting we make it illegal to vote, it is merely an example of a system where there is an incentive to vote (symmetrically, a negative incentive to not vote). We would presumably do it by rewarding voters instead.

Reply:
> d legal procedure if you are to punish, otherwise that is dystopian. Forced political participation upholds established systems rather than allows for people to disengage from a broken system and begin to form new ones. It is anti-revolutionary and will stymie growth. With legal procedure, at least one can have the opportunity to bring up why they did not vote if they are forced to vote and more law in an ecosystem can be created.
> 
> > Well, several governments including Australia make it illegal not to vote, which is an incentive.

Well I am suggesting you can make it illegal not to vote and I wish to have your feedback if we want to discuss this more. As long as we have good legal procedure we can do this if we get more community engagement. I am also suggesting we should have a legal system in place to address negative behavior in an ecosystem and a legal system in place to encourage positive behavior. This would be the legal authority that grants the power to reward as you suggest.